src/publish.ts

```diff
@@ -55,12 +61,18 @@ const publish = async ({ extensionId, target, asset }: PluginConfig) => {
   if (!publishRes.status.includes('OK')) {
     const errors: SemanticReleaseError[] = []
     for (let i = 0; i < publishRes.status.length; i += 1) {
-      const message = publishRes.statusDetail[i]
       const code = publishRes.status[i]
-      const err = new SemanticReleaseError(message, code)
-      errors.push(err)
+      const message = publishRes.statusDetail[i]
+      if (errorWhitelist.includes(code)) {
+        logger.log(`${code}: ${message}`)
```

Comment:
Maybe we should use `warn` instead, I think it would raise more visibility for the message...

```suggestion
        logger.warn(`${code}: ${message}`)
```